06 - poslowie/poslowie.js

```diff
@@ -0,0 +1,28 @@
+const poslowieApi = new URL("https://api-v3.mojepanstwo.pl/dane/poslowie/");
+const okreg = "conditions[poslowie.sejm_okreg_id]";
+const krakow = 13;
+
+class Rebels{
+    constructor(api, conditions, param) {
+        this.api = api;
+        this.conditions = conditions;
+        this.param = param;
+    }
+    
+   	displayRebels(data) {
```

Comment:
please fix formatting in this file - in VSC you can do it with one action on the whole file